lib/opentelemetry-proto/src/convert.rs

```diff
@@ -78,69 +80,144 @@ fn kv_list_into_value(arr: Vec<KeyValue>) -> Value {
     )
 }
 
-impl From<ResourceLog> for Event {
-    fn from(rl: ResourceLog) -> Self {
-        let mut le = LogEvent::default();
+// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.15.0/specification/logs/data-model.md
+impl ResourceLog {
+    fn into(self, log_namespace: LogNamespace, now: DateTime<Utc>) -> Event {
```

Comment:
Can you just call this `into_event`? Having an inherent method named after a common trait method, like `into`, is confusing.

Reply:
Resolved in https://github.com/vectordotdev/vector/pull/15469/commits/b424739780a0b45ef32d2bd51570537c1391bca7